### PR TITLE
Remove "Install preview channel" Step from GetStart & Sample articles

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered articles for review include:
* https://review.learn.microsoft.com/microsoft-edge/webview2/get-started/wpf?branch=pr-en-us-3241
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/get-started/wpf
* https://review.learn.microsoft.com/microsoft-edge/webview2/samples/webview2wpfbrowser?branch=pr-en-us-3241
   * Before/live: https://learn.microsoft.com/microsoft-edge/webview2/samples/webview2wpfbrowser

AB#52971619

### CODE REVIEW

File: microsoft-edge/webview2/samples/webview2browser.md [hunk 1]
```diff
@@ -37,27 +37,21 @@ This sample is built as a Win32 [Visual Studio 2019](https://visualstudio.micros
...
-
-
-<!-- ====================================================================== -->
-## Step 2: Install Visual Studio
+## Step 1: Install Visual Studio
 
 1. Install [Visual Studio](https://visualstudio.microsoft.com/vs/), including C++ support.
 
-The [WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/) is recommended for the installation.<!-- TODO: give actionable steps/instructions.  Why preview channel and also Runtime? -->  The minimum version is 86.0.622.38.
+The [WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/) is recommended for the installation.<!-- TODO: give actionable steps/instructions -->  The minimum version is 86.0.622.38.
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/samples/webview2browser.md:44]: I think we can remove this line. The other sample apps don't have this line, and most Windows versions will now come with WebView2 installed

Comment (victorhuangwq) [inline microsoft-edge/webview2/samples/webview2browser.md:44]:
```suggestion
```

**Review by victorhuangwq — CHANGES_REQUESTED** [review]
Small change requested, but otherwise looks good to me.

File: microsoft-edge/webview2/samples/webview2browser.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/samples/webview2browser.md:44]: thx, was wondering, removed

Comment (mikehoffms) [reply microsoft-edge/webview2/samples/webview2browser.md:44]: thx, was wondering, removed

**Review by victorhuangwq — APPROVED** [review]
LGTM

**Pull request merged**